code-challenges.test.js

```diff
@@ -23,11 +35,42 @@ const fibonacciLength2 = 10
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
-
+// Name: fibSeq
+// Input: Number
+// Output: an array containing the Fibonacci sequence.
+// Process: create a function that iterates through a array using a for loop.
+//And then add the added value to the end of the array using .push()
+
+//***(Not gonna lie I barely understand this Fibonacci sequence. I had to look it up)
+// I was having some issues with this first process
+// I went and made a more simpler solution. I do realize my return was in the wrong code block with the first attempt
+
+// const fibSeq = (n) => {
+//   const fibArr = [1, 1]
+//   for(let i = 2; i< n; i++){
+//    const nextNum = fibArr[i-2] + fibArr[i-1]  
+//     fibArr.push(nextNum)
+//     return fibArr
+//   }
+// }
+
+  const fibSeq = (num) => {
+    fib = [1, 1]
+    for(let i = 2; i < num; i++){
+      fib[i] = fib[i-2] + fib[i-1] 
```

Comment:
Each iteration is assigning the current index value to the sum of the previous 2 index values. So on the first iteration, we start at index 2. The code is saying that the value at index 2 (third number) is the sum of the value at index 1 (second number) and index 0 (first number). And then the value of `i` increases by 1 each iteration.